ui/entry-perf.js

```diff
@@ -10,6 +10,7 @@ require('font-awesome/css/font-awesome.css');
 require('./css/treeherder-navbar.css');
 require('./css/perf.css');
 require('./css/treeherder-loading-overlay.css');
+require('./vendor/css/metricsgraphics.css');
```

Comment:
Please can you use the npm package rather than vendoring this?
https://www.npmjs.com/package/metrics-graphics

IMO we should have a "no new vendored packages" policy for Treeherder (with exceptions needing approval first).